### PR TITLE
Fix the spelling mistake in the git commit message

### LINKED ISSUE #91: Fix the spelling mistake in the Git Commands section in readme.md file
**Describe the bug**
Currently, there is a spelling mistake in the Git Commands section in the readme.md file

**To Reproduce**
1. Go to 'readme.md'
2. Scroll down to 'Git Commands'
3. See error

**Expected behavior**
Fix the spelling mistake in the Git Commands section in readme.md file

- It should be `git commit -m 'Your Message'`

**Screenshots**
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by Prasadkpd**
## Purpose

The purpose of this Pull Request is to fix #91

## Goals

Fixing the spelling mistake in the git commit message in readme.md file

## Approach

Fixed the spelling mistake in the git commit message in readme.md file

### Screenshots

  
image

### Preview Link

https://github.com/Prasadkpd/carx-landing-page/tree/fixSpell

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ git status
 git add .
 ```
 ```
-git commit -m 'Your Massege'
+git commit -m 'Your Messege'
```
Comment (YoshithaRathnayake) [inline README.md]: It should be `Message`
```suggestion
git commit -m 'Your Message'
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (YoshithaRathnayake) [reply README.md]: @Prasadkpd

Comment (Prasadkpd) [reply README.md]: Done

**Review by YoshithaRathnayake — APPROVED** [review]
Thank you @Prasadkpd 
Can you merge this @sathsarabandaraj

**Pull request merged**